stapler/stapler.go

```diff
@@ -0,0 +1,529 @@
+// package stapler implements OCSP stapling feature described here: http://en.wikipedia.org/wiki/OCSP_stapling
```

Comment:
You might also want to reference the two RFC that deal with OCSP:

* https://tools.ietf.org/html/rfc2560
* http://tools.ietf.org/html/rfc6066